src/FakeItEasy/Core/FakeManager.cs

```diff
@@ -93,7 +94,7 @@ public virtual IEnumerable<ICompletedFakeObjectCall> RecordedCallsInScope
             get { return FakeScope.Current.GetCallsWithinScope(this); }
         }
 
-        internal List<ICompletedFakeObjectCall> AllRecordedCalls
+        internal IEnumerable<ICompletedFakeObjectCall> AllRecordedCalls
         {
             get { return this.recordedCallsField; }

```

Comment:
This is a minor point, bordering on a nitpick, but with the addition of `RecordCall()` below, and the switch to `IEnumerable<T>` from `List<T>`, it looks like we're shifting more towards hiding the backing field as an implementation detail.

For completeness, I'd probably change this to:

```C#
internal IEnumerable<ICompletedFakeObjectCall> GetRecordedCalls()
{
    return this.recordedCallsField.ToList();
}
```

The call to `ToList()` is to prevent access to the backing field, ensuring that all mutation happens via `RecordCall()`.

As part of this refactoring I'd probably also rename `recordedCallsField` to `recordedCalls` (presumably the suffix is something left over from some old code state).